app/lib/common/utils/utils.dart

```diff
@@ -120,12 +120,22 @@ String getDayFromDate(UtcDateTime utcDateTime) {
   return day;
 }
 
-String jiffyTime(int timeInterval) {
+String getTimeFromDate(BuildContext context, UtcDateTime utcDateTime) {
+  final localDateTime = toDartDatetime(utcDateTime).toLocal();
+  return DateFormat(getLocalisedTimeFormat(context)).format(localDateTime);
+}
+
+String getLocalisedTimeFormat(BuildContext context) {
+  bool is24HoursFormat = MediaQuery.of(context).alwaysUse24HourFormat;
+  return is24HoursFormat ? 'HH:mm' : 'hh:mm a';
+}
+
+String jiffyTime(BuildContext context, int timeInterval) {
   final jiffyTime = Jiffy.parseFromMillisecondsSinceEpoch(timeInterval);
   final now = Jiffy.now().startOf(Unit.day);
   if (now.isSame(jiffyTime, unit: Unit.day)) {
-    // (00:00 AM/PM)
-    return jiffyTime.jm;
+    bool is24HoursFormat = MediaQuery.of(context).alwaysUse24HourFormat;
+    return is24HoursFormat ? jiffyTime.Hm : jiffyTime.jm;
```

Comment:
this seems like a rather convoluted way of doing
`DateFormat.jm().format(localDateTime));` (from the `intl` package we already have included anyways) 

Any reason for not just using [`Intl.DateFormat`](https://api.flutter.dev/flutter/intl/DateFormat-class.html)?